api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -588,9 +588,8 @@ public void getGlobalProperty_shouldFailIfUserHasNoPrivileges() {
 		Context.logout();
 		Context.authenticate(getTestUserCredentials());
 		
-		APIException exception = assertThrows(APIException.class, () -> adminService.getGlobalProperty(property.getProperty()));
-		assertEquals(exception.getMessage(), String.format("Privilege: %s, required to view globalProperty: %s",
```

Comment:
Do you mind explaining why you changed the contents of ` String.format()`?

Reply:
The format of the exception message changes after adding the authorization.